.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Codacy found an issue: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6178416)